odoo/addons/base/models/ir_qweb.py

```diff
@@ -424,10 +424,10 @@ def _compile_expr(self, expr):
     def _get_attr_bool(self, attr, default=False):
         if attr:
             if attr is True:
-                return ast.Name(id='True', ctx=ast.Load())
+                return ast.Constant(True)
             attr = attr.lower()
             if attr in ('false', '0'):
-                return ast.Name(id='False', ctx=ast.Load())
+                return ast.Constant(False)
             elif attr in ('true', '1'):
-                return ast.Name(id='True', ctx=ast.Load())
-        return ast.Name(id=str(attr if attr is False else default), ctx=ast.Load())
+                return ast.Constant(True)
+        return ast.Constant(attr if attr is False else default)
```

Comment:
wrapping `default` in `bool()` might be a good idea just to play it safe (though not really important, probably).